packages/terra-demographics-banner/src/DemographicsBanner.jsx

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable global-require, import/no-dynamic-require, react/no-unused-prop-types */
```

Comment:
We can do [eslint-disable-next-line](https://github.com/cerner/terra-core/blob/7f1d3732290e7335506c5a630e2ac616361cd9ff/packages/terra-site/src/examples/button/ButtonOnClick.jsx#L5) on line 84 so it will continue to lint the rest of the file.